config/webhook/manifests.yaml

```diff
@@ -89,9 +89,9 @@ webhooks:
     service:
       name: webhook-service
       namespace: system
-      path: /mutate-redhatcop-redhat-io-v1alpha1-databasesecretengineconfig
+      path: /mutate-redhatcop-redhat-io-v1alpha1-gcpauthengineconfig
   failurePolicy: Fail
-  name: mdatabasesecretengineconfig.kb.io
```

Comment:
also these substitutions look suspicious, please regenerate this file with `make manifests`